libqtile/scripts/qtile_top.py

```diff
@@ -101,22 +101,22 @@ def get_stats(scr, client, group_by='lineno', limit=10, seconds=1.5,
             line = linecache.getline(frame.filename, frame.lineno).strip()
             if line:
                 code = line
-            mem = "{0:.1f} KiB".format(stat.size / 1024)
-            filename = "{0:s}:{1:s}".format(filename, frame.lineno)
-            scr.addstr(cnt + 1, 0, '{0:<3s} {1:<40s} {2:<30s}'.format(index, filename, mem))
+            mem = "{:.1f} KiB".format(stat.size / 1024.0)
```

Comment:
Ah, we should probably do the standard `__future__` imports for stuff like division.